tests/Unit/IO/Importers/Test_VolumeDataReaderActions.cpp

```diff
@@ -120,14 +109,23 @@ SPECTRE_TEST_CASE("Unit.IO.Importers.VolumeDataReaderActions", "[Unit][IO]") {
                                               {1, {{{1, 0}, {2, 3}}}},
                                               {1, {{{1, 0}, {5, 4}}}},
                                               {0, {{{1, 0}, {1, 0}}}}};
+  std::unordered_map<ElementId<2>,
```

Comment:
Please explain so I understand what these changes are doing?

Reply:
In this test I didn't move the read-in data into the DataBox, but had the sample data in the DataBox and compared both in the callback action. Now I begin with an empty DataBox, have the new `ReceiveVolumeData` action populate it with the read-in data and compare it to the sample data that lives outside (in the test case).